packages/expo-video/src/VideoView.web.tsx

```diff
@@ -48,8 +55,36 @@ export const VideoView = forwardRef((props: { player?: VideoPlayer } & VideoView
     exitFullscreen: () => {
       document.exitFullscreen();
     },
+    startPictureInPicture: () => {
+      videoRef.current?.requestPictureInPicture();
+    },
+    stopPictureInPicture: () => {
```

Comment:
I think our types are not correct for these functions – they are all async on iOS, so the return type should be `Promise<void>` or, in other words, they should be async functions. Then you can use `try..catch` with `await` instead.